src/cpu/reorder/simple_sparse_reorder.hpp

```diff
@@ -100,7 +100,7 @@ struct simple_sparse_reorder_impl<SIMPLE_SPARSE_REORDER_TEMPL_CALL,
 
     static size_t get_scratchpad_size(const memory_desc_wrapper &input_d,
             const memory_desc_wrapper &output_d) {
-        const auto nelems = static_cast<size_t>(output_d.nelems(true));
```

Comment:
`nelems` can only return < 0 when one of the dimensions is runtime because this is how wildcard defined: https://github.com/oneapi-src/oneDNN/blob/c07fe05d9c63d5a316773d055888844218931702/include/oneapi/dnnl/dnnl_types.h#L2279

This reorder doesn't support runtime dimensions so we return unimplemented before we can reach this place so it is safe to keep `size_t`.